smart-contracts/published-npm-modules/V1.2/README.md

```diff
@@ -1,11 +1,59 @@
 # Unlock ABI v1.2
 
 This npm module includes the ABI for the smart contracts Unlock.sol and PublicLock.sol for v1.2
-The smart contracts have been compiled as of commit b7656b08.
+The smart contracts have been compiled as of commit 0bea7a00.
 
 Changelog:
 
 ```
+* 0bea7a00 - Implementing disabling transfers (#5303) (2 days ago)
```

Comment:
I do wonder if we should strip out the commit age info, because it goes stale and doesn't actually agree with previous entries, because it's relative to when the commit was made.

Reply:
oh yes. let me do that!